src/components/RawQueryEditor.tsx

```diff
@@ -73,23 +86,27 @@ export const RawQueryEditor: React.FC<RawQueryEditorProps> = (props) => {
         return model && kusto(model.uri);
       })
       .then((worker) => {
-        if (schema) {
-          // Populate Database schema with macros
-          Object.keys(schema.Databases).forEach((db) =>
-            Object.assign(schema.Databases[db].Functions, getFunctions(templateSrv.getVariables()))
-          );
-          worker?.setSchemaFromShowSchema(schema, 'https://help.kusto.windows.net', props.database);
-        }
+        setWorker(worker);
       });
   };
 
+  useEffect(() => {
+    if (worker && schema) {
+      // Populate Database schema with macros
+      Object.keys(schema.Databases).forEach((db) =>
+        Object.assign(schema.Databases[db].Functions, getFunctions(templateSrv.getVariables()))
```

Comment:
Put result in `templateSrv.getVariables()` in a variable so we don't have to call it multiple times?